genai-perf/README.md

```diff
@@ -407,13 +407,21 @@ URL of the endpoint to target for benchmarking. (default: `None`)
 
 ##### `-b <int>`
 ##### `--batch-size <int>`
+##### `--batch-size-text <int>`
 
-The batch size of the requests GenAI-Perf should send.
+The text batch size of the requests GenAI-Perf should send.
 This is currently only supported with the
-[embeddings](docs/embeddings.md), image_retrieval, and
+[embeddings](docs/embeddings.md) and
 [rankings](docs/rankings.md) endpoint types.
 (default: `1`)
 
+##### `--batch-size-images <int>`
```

Comment:
nit but maybe `--batch-size-image` to be consistent with the text counterpart?

Reply:
Done!